doc/nrf/app_memory.rst

```diff
@@ -84,6 +84,44 @@ Besides applying `General recommendations`_, you can also complete the following
 * Reduce the verbosity of assert messages by setting :kconfig:option:`CONFIG_ASSERT_VERBOSE` to ``n``.
 * Check `Thread`_ memory footprint optimization actions, as the Matter application layer uses OpenThread.
 
+.. _app_memory_nfc:
+
+NFC
+***
+
```

Comment:
Just please add an introductory line after the heading. Maybe like the following one?

```suggestion

The :ref:`ug_nfc` protocol lets you optimize memory footprint for both the tag and the poller roles.

```

Reply:
Added